themes/src/main/resources/theme/base/admin/resources/partials/client-detail.html

```diff
@@ -37,7 +37,7 @@
                 </div>
                 <kc-tooltip>{{:: 'client.enabled.tooltip' | translate}}</kc-tooltip>
             </div>
-            <div class="form-group clearfix block">
+            <div class="form-group clearfix block" data-ng-show="protocol != 'docker-v2'">
                 <label class="col-md-2 control-label" for="consentRequired">{{:: 'consent-required' | translate}}</label>
                 <div class="col-sm-6">
                     <input ng-model="clientEdit.consentRequired" name="consentRequired" id="consentRequired" onoffswitch on-text="{{:: 'onText' | translate}}" off-text="{{:: 'offText' | translate}}"/>
```

Comment:
So I took a look at this - other than one-off hacking the order (which I don't think we wanna do), this would require a pretty broad change to the data model for the serverInfo's ProviderRepresentation.  Right now angular is just doing a naive sort:

`$scope.protocols = Object.keys(serverInfo.providers['login-protocol'].providers).sort();`

Could we perhaps create an issue and do that on a follow-up review?  This one's already gotten pretty big, and I'd consider that a more general aesthetic fix.  For instance, there is not a determinitive order in authenticator choices in the dropdown, etc.  I think the angular models could use a widespread application of some kind of salience/display order.